pkg/config/types/parse.go

```diff
@@ -12,7 +12,8 @@ func CastConfigValueForKey(key string, value any) (any, error) {
 	key = strings.ToLower(key)
 	typ, ok := AllKeys()[key]
 	if !ok {
-		return nil, fmt.Errorf("%q is not a valid config key", key)
+		return nil, fmt.Errorf("%q is not a valid config key. "+
+			"Run 'bacalhau config list' for the compelte list of valid config keys", key)
```

Comment:
nit: use os.Args[0] instead of `bacalhau` incase using a different binary path or alias

Reply:
good idea, done.